docs/src/modules/spring/pages/actions.adoc

```diff
@@ -0,0 +1,78 @@
+= Implementing Actions in Spring 
+:page-supergroup-java-scala: Language
+
+include::ROOT:partial$include.adoc[]
+include::partial$actions.adoc[]
+
+In this first example, you will learn how to implement an Action as a pure stateless function. You will create a `FibonacciAction` that takes a number and returns the
+next number in the Fibonacci series.
+
+== Implementing the Action
+
+To implement this action you need the following:
+
+- Extend our class from `kalix.javasdk.action.Action`. This is generic. No matter what action you want to create you always need to extend from link:{attachmentsdir}/testkit/kalix/javasdk/action/Action.html[`Action` {tab-icon}, window="new"]. The rest is specific to this case
+- Add the Spring annotation @RequestMapping to provide a REST endpoint for the function. Here the stateless function should be reachable via HTTP.
+- Add the Spring annotations @GetMapping and @PostMapping to provide paths for GET and POST to calculate the Fibonacci of a number. Both functions do the same thing and implementation-wise the function exposed with GET calls the function exposed with POST.  
+
+
+[source,java,indent=0]
+.src/main/java/com/example/fibonacci/FibonacciAction.java
+----
+include::example$spring-fibonacci-action/src/main/java/com/example/fibonacci/FibonacciAction.java[tag=implementing-action]
+----
+<1> `isFibonacci` checks if a number is a Fibonacci number
+<2> `nextFib` calculates the next number.
+<3> This `nextNumber` implementation calls the `nextNumber` implementation below.
+<4> The `nextNumber` implementation first checks if the input number belongs to the Fibonacci series. If so, it calculates the
+next number and builds a reply using `effects().reply()`.
+<5> Otherwise, if the input number doesn't belong to the Fibonacci series, it builds an `Effect` reply error.
+
+Actions return effects, that is, `Action.Effect`. You have different types of effects. A reply, an error, a forward to the call to another component, and to all of those you can add side effects. Here you want only the result of the calculation or an error. Therefore you are using `.reply` and `.error`.
+
+=== Multiple replies / reply streaming
+
+An Action may return a stream of integers. To do this you need to define the return type as `reactor.core.publisher.Flux<Effect<Integer>>`. 
+
+The stream may publish an arbitrary number of replies.
+
+ifdef::todo[TODO: add a streamed Fib series calculation]
+
+
+== Registering the Action
+
+TODO: create Main docs and add partial here
```

Comment:
there is no action registration. The Spring SDK does it automatically by classpath scanning.

Reply:
Nice catch. I forgot to remove the title although I wanted to explain how to set up the Main here. How about I create a partial and we later use it in other parts of the documentation? 
Or I skip entirely this point here?

Reply:
I ended up just removing that section from the EventSourced page.. I think somewhere else we should explain that the components are auto registered...

Reply:
I would rather don't explain auto registering anywhere. The user would be better off don't knowing anything about it I think. 
I thought this placeholder to explain how a `Main` is created so the user has a view of the whole application, with the entry point included.  I guess my question is how you feel about it guys? Do you think we should explain in Action, Value Entity, and Event Sourced Entity how to create the Main that starts the app? and if so and it is the same in all of them, should we create a partial for it and reuse it?

Reply:
No, we should not explain it because users should not create it. They should start from the archetype and the archetype comes with it already correctly configured. 

I added a blank page about dependency injection. We do need to explain that components get picked by Spring Boot and that users can build components and let Spring wire dependencies into it.

Reply:
ok. I'll remove